src/components/EmergencyShutoffCard/index.jsx

```diff
@@ -57,8 +58,8 @@ function EmergencyShutoffCard({
         title={
           <Typography component="h2" variant="h6">
             {emergencyShutoff.scheduledChange
-              ? 'Updates are scheduled to be enabled for this product and channel'
-              : 'Updates are currently disabled for this product and channel'}
+              ? `Updates are scheduled to be enabled for the ${product} ${channel} channel`
+              : `Updates are currently disabled for the ${product} ${channel} channel`}
```

Comment:
I wonder what would be outputted when a filter has no channel (e.g., CDM)

Reply:
Shouldn't be an issue, because the Emergency Shutoffs require a Product and Channel (arguably we should change this, but that's how it is now), and we don't enable the UI unless both are present for the current filter: https://github.com/mozilla-frontend-infra/balrog-ui/pull/184/files#diff-6bdfeacc9dead52092c38bbdc3958037R1153